core/src/main/scala/org/typelevel/paiges/Doc.scala

```diff
@@ -212,9 +212,9 @@ sealed abstract class Doc extends Product with Serializable {
    * longer than `width` -- it just attempts to keep lines within this
    * length when possible.
    */
-  def render(width: Int): String = {
+  def render(width: Int, trim: Boolean = Doc.DEFAULT_TRIM): String = {
```

Comment:
I seem to recall @johnynek would rather have 2 methods instead of 1 with default args because of binary compatibility. I don't see his comments on this PR, but was that discussed at all?